style-guide/style_guide.md

```diff
@@ -1,15 +1,125 @@
+
+## Syntax of `index.md` file
+
+The `index.md` file is a markdown file with YAML-formatted metadata at the
+top, delimited with `---`, followed by [content](#content) that serves as
+instructor notes for the node.
+
+### Metadata
+
+For a complete description of YAML formatting for metadata, see
+[the YAML specification](http://www.yaml.org/spec/1.2/spec.html).  This
+section defines specific metadata and its meaning.  The metadata components
+may appear in any order.
+
+#### Required Metadata
+
+Everynode is required to have the following metadata:
+
+* _title_: A human readable title for the node.  Titles should be unique,
+  descriptive, short, and capitalized.
+
+  e.g. `title: A NECX Node Template`
+
+* _uuid_ : A unique universal identifier formatted as words joined with
+  hyphens, and often similar to the title.  UUIDs should be unique,
+  descriptive, short, and lowercase.
+
+  e.g. `uuid: necx-node-template`
+
+#### Required Metadata for Internal Nodes
+
+The following metadata are required for all internal nodes, and its presence
+identifies the node as an internal node rather than a boundary node.
+
+* _learning objective_: A phrase indicating the learning objective of the
+  node.  Each node is expected to have a single learning objective.
+
+  e.g.
+```
+learning objective:
+   construct a node with the correct syntax and contents
+```
+
+* _prerequisites_: A list of node uuids that are considered prerequisite
+  knowledge for this node.  All prerequisites must be present in the exchange,
+  whether as other interior nodes or as boundary nodes.
+
+  e.g.
+```
+prerequisites:
+    - other-content
+    - another-node
+```
+
+* _assessments_: A list of assessment activities that allow a student to
+  demonstrate their compitency with the learning objective.  Each assessment
+  is stored in a separate subdirectory within he `assessments` subdirectory.
+  The name of that subdirectory should be a uid, but only unique within this
+  node.  The entries in the metadata list are the uid subdirector names.
+  There must be at least one assessment.  Individual assessments may take on
+  [many formats](assessement-formats.md).
+
+  e.g.
+```
+assessments:
+  - a-multiple-choice-question
+  - a-notebook-problem
+  - write-an-essay
+```
+
+### Content
+
+Markdown formatted content files for each node contain the lecture
+material/instructor notes for the subject matter.
 
 Syntax for the file should adhere to the following guidelines:
 1. No html tags
 2. Use latex tags for equations; i.e., $e^{-\lambda t}$ or $$e^{-\lambda t}$$
-3. Images should be stored in an img subdirectory within the parent node directory
+3. Images should be stored in an `img` subdirectory within the parent node directory
+
+
+## Additional Elements
 
-#### Exercises
+In addition to the required node elements, many elements are recommended and
+should be incorporated into the `index.md` file as metadata, links,
+references, or otherwise.
 
+- Text, equations
```

Comment:
I would suggest "Nomenclature" or the equivalent.  This might actually be something desired as part of the overall style guide, too.  We may all agree on $\Sigma$ for a macroscopic cross section, but is it $\Sigma_{sgg'}$, $\Sigma_{sg\gets g'}$, $\Sigma_{sg'\to g}$, etc.